modules/gitops-configuring-tls-for-redis-with-autotls-disabled.adoc

```diff
@@ -0,0 +1,222 @@
+// Module is included in the following assemblies:
+//
+// * /cicd/gitops/configuring-secure-communication-with-redis.adoc
+
+[id="gitops-configuring-tls-for-redis-with-autotls-disabled_{context}"]
+= Configuring TLS for Redis with autotls disabled
+
+.Procedure
+
+. Create an Argo CD instance:
+
+.. In the *Administrator* perspective of the web console, use the left navigation panel to go to *Administration* -> *CustomResourceDefinitions*.
+
+.. Search for `argocds.argoproj.io` and click `ArgoCD` custom resource definition (CRD).
+
+.. Click *Instances* -> *Create ArgoCD*.
+
+.. In the YAML field, replace the YAML code:
++
+.Example ArgoCD CR with autotls disabled
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd <1>
+  namespace: openshift-gitops <2>
+spec:
+  ha:
+    enabled: true <3>
+----
++
+<1> Specify a name of the Argo CD instance. In this example, the name is `argocd`.
+<2> Specify a namespace where you want to run the Argo CD instance. In this example, the namespace is `openshift-gitops`.
+<3> This enables the HA feature. If you do not want to enable HA, do not include this line or put the value as `enabled: false`.
++
+[NOTE]
+====
+The setting with HA enabled requires a cluster with at least three worker nodes.
+====
+
+.. Click *Create*.
+
+.. Verify that the Argo CD pods are ready and running. For the Argo CD instance with HA enabled, this might take a few minutes.
++
+[source,terminal]
+----
+$ oc get pods
```

Comment:
```suggestion
$ oc get pods -n openshift-gitops
```

Reply:
Hello @varshab1210 as I did not want to create commands for specific namespaces, I incorporated your suggestion by adding
`-n <namespace>`
I hope you are okay with that change. I added it to the places you suggested. Please let me know if the PR looks good to you now. Thank you!